examples/functions/golang/pool/mmf/matchfunction.go

```diff
@@ -20,12 +20,11 @@
 package mmf
 
 import (
-	structpb "github.com/golang/protobuf/ptypes/struct"
 	"github.com/rs/xid"
 	"open-match.dev/open-match/examples"
 	mmfHarness "open-match.dev/open-match/pkg/harness/function/golang"
-
 	"open-match.dev/open-match/pkg/pb"
+	"open-match.dev/open-match/pkg/structs"
```

Comment:
Aha - now that I see it, I do have a naming suggestion. 

Given that this is very much in context of a property only, how about call it property? 
"open-match.dev/open-match/pkg/property" (or properties) seems to imply the property helpers better than:
"open-match.dev/open-match/pkg/structs" 
What say?

Reply:
after some discussion, keeping it as structs.